### ISSUE TITLE
Bug: Kint Config isnt loaded

### ISSUE THREAD

**Issue by fapth** [codeigniter4/CodeIgniter4#2830]
**Describe the bug**
After upgrading CI4.0.0rc to CI4.0.2 i'm getting this error when i call a simple side. I could fix it, when i added the Kint - Config from "vendor\codeigniter4\framework\app\Config" to "app\Config"

Error Message:
fapth@DESKTOP-Q52SKBB:/mnt/j/CI_Framework/public$ php index.php home index
An uncaught Exception was encountered

Type:        ErrorException
Message:     Trying to get property 'maxDepth' of non-object
Filename:    /mnt/j/CI_Framework/vendor/codeigniter4/framework/system/CodeIgniter.php
Line Number: 232

	Backtrace:
						-232 - /mnt/j/CI_Framework/vendor/codeigniter4/framework/system/CodeIgniter.php::errorHandler
								-193 - /mnt/j/CI_Framework/vendor/codeigniter4/framework/system/CodeIgniter.php::initializeKint
								-168 - /mnt/j/CI_Framework/vendor/codeigniter4/framework/system/bootstrap.php::initialize
								-36 - /mnt/j/CI_Framework/public/index.php::require
			

**CodeIgniter 4 version**
CodeIgniter 4.0.2

**Affected module(s)**
Which package or class is the bug in, if known.

**Expected behavior, and steps to reproduce if appropriate**
A clear and concise description of what you expected to happen,
and how you got there.  
Feel free to include a text/log extract, but use a pastebin facility for any
screenshots you deem necessary.

**Context**
 - OS: Linux on Windows 10 / Ubuntu 18.04
 - PHP version 7.2

**Comment by agungsugiarto** [comment]
You need self update config kint in your appstarter. See in history commit from repository [appstarter](https://github.com/codeigniter4/appstarter)

**Comment by agungsugiarto** [comment]
Or look my commit from sample project after update to last version [sample update](https://github.com/agungsugiarto/codeigniter4-ajax-crud/commit/eb3328cfb30b25bd85d8ec550d1a2256438a2931)

**Comment by MGatner** [comment]
Yes as @agungsugiarto says you need to copy in the Kint Config. Changes to App will always require intervention as described by the docs.
